home.html

```diff
@@ -26,17 +33,17 @@
   </head>
 
   <body>
-    <div class="container">
+    <div class="container-main">
```

Comment:
- [ ] [OPTIONAL] It would be nice if for all your sections you can use `min-height` CSS style to make it more responsive (also using 100vh) for instance:
```
section {
  min-height: 100vh;
}
```